arraycontext/impl/pytato/compile.py

```diff
@@ -642,8 +631,10 @@ def __call__(self, arg_id_to_arg) -> ArrayContainer:
         from .utils import get_cl_axes_from_pt_axes
         from arraycontext.impl.pyopencl.taggable_cl_array import to_tagged_cl_array
 
+        fn_name = self.pytato_program.program.entrypoint
```

Comment:
This and the `fn_name`s below appear to be inconsistent. Is there a specific reason for this?

Reply:
Thanks, this is hopefully fixed in b1c3a73. Part of the challenge here is that `CompiledPyOpenCLFunctionReturningArray` appears to be unused.